### DISCUSSION TITLE
Say hello to the UI!

### DISCUSSION THREAD

**Discussion by sr229** [vignetteapp/vignette#101] [Announcements]
As part of the sweeping changes as we finally resume to developing after the Typhoons that hit the Philippines, I've finalized the UI we will be using for the entire project!

[Figma Link](https://www.figma.com/file/LHUBNtioxj1n8okt6L234D/Mockv4-UI?node-id=0%3A1)

![image](https://user-images.githubusercontent.com/14976516/101280183-9ffa4080-3802-11eb-958d-182b081b5b11.png)
![image](https://user-images.githubusercontent.com/14976516/101280185-a7b9e500-3802-11eb-99d2-8155e0772aa2.png)
![image](https://user-images.githubusercontent.com/14976516/101280189-adafc600-3802-11eb-897a-42b407f47aea.png)

Got any feedback? Let us know by commenting below.

**Comment by katsuko9** [comment]
looks nice! one thing that I'd like is having the leftmost and uppermost column of the window more compact (or by offering an option to).
dark mode is always a nice addition.
(-ω-ゞ

**Reply by sr229** [reply]
> looks nice! one thing that I'd like is having the leftmost and uppermost column of the window more compact (or by offering an option to).
> dark mode is always a nice addition.
> (-ω-ゞ

That's something I would envision to implement, for now this was designed in Low DPI since it's a baseline of how it'd look like on smaller ratios.
